src/main/java/com/github/hokkaydo/eplbot/module/code/JavaRunner.java

```diff
@@ -0,0 +1,112 @@
+package com.github.hokkaydo.eplbot.module.code;
+
+import javax.tools.ToolProvider;
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.IOException;
+import java.io.PrintStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.ScheduledThreadPoolExecutor;
+import java.util.concurrent.TimeUnit;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+public class JavaRunner implements Runner{
+    private static final String OUTPUT_PATH = System.getProperty("user.dir")+"\\src\\temp\\";
+    private static final ScheduledExecutorService SCHEDULER = new ScheduledThreadPoolExecutor(1);
+
+    @Override
+    public String run(String input, Integer runTimeout) {
+        if (!input.equals(safeImports(input))){
+            return "Invalid imports";
+        }
+        String className = regexClassName(input);
+        writeFile(input,Path.of(OUTPUT_PATH+"\\"+className+".java"));
+        String filePath = OUTPUT_PATH + File.pathSeparator + className + ".java";
+        try {
```

Comment:
No need for this big enclosed try-catch, especially with a stinky "Exception" catching.
Only the InputStream#read method needs error handling, just enclose the while loop with it